react/features/base/tracks/functions.js

```diff
@@ -105,7 +105,26 @@ export function getLocalAudioTrack(tracks) {
  * @returns {(Track|undefined)}
  */
 export function getLocalTrack(tracks, mediaType) {
-    return tracks.find(t => t.local && t.mediaType === mediaType);
+    return getLocalTracks(tracks).find(t => t.mediaType === mediaType);
+}
+
+/**
+ * Returns local track by media type.
```

Comment:
this comment seems to apply to the function above.

Reply:
yes, will fix that, looks like I've just copy pasted instead of updating